referral requests/README.md

```diff
@@ -0,0 +1,173 @@
+        }
+    ]
+}
+```
+</details>
+
+## JSON FHIR > XML HL7
+
+An Allergy Intolerance is mapped to a RequestStatement 
+
+| Mapped to (XML HL7)                                                                                                          | Mapped from (JSON FHIR / other source )                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `RequestStatement / id [@root]`                                                                                              | `ReferralRequest.id` or generated UUID                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `RequestStatement / code [@code='3457005'] [@displayName='Patient referral'] [@codeSystem='2.16.840.1']`                     | when `ReferralRequest.reasonCode` is missing                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Do you think this might be better in a footnote? The code system is `2.16.840.1.113883.2.1.3.2.4.15` (SNOMED). We could just say if reasonCode isn't present then the adaptor inserts a default SNOMED code of `3457005` (Patient referral)

Reply:
Yep fair comment, I was a bit unsure of how to format it, but in hindsight a footnote does seem like a better idea.